docs.md

```diff
@@ -0,0 +1,1366 @@
+
+```
+What is the right answer?
+
+[[right answer]]
+<script>
+  let input = "@input".trim().toLowerCase();
+  input == "right answer" || input == "correct answer";
+</script>
+***
+<div class = "answer">
+
+For this question, either "right answer" or "correct answer" (not case sensitive) counts as correct.
+
+</div>
+***
+```
+
+What is the right answer?
+
+[[right answer]]
+<script>
+  let input = "@input".trim().toLowerCase();
+  input == "right answer" || input == "correct answer";
+</script>
+***
+<div class = "answer">
+
+For this question, either "right answer" or "correct answer" (not case sensitive) counts as correct.
+
+</div>
+***
+
+This question accepts any of several items from a list of possible correct answers. It is not case sensitive (that's the little `i` at the end of the regex).
+
+With flexible answers like this, it's definitely a good idea to include a follow-up to help the learner put their answer in context.
+
+For example, if the question was "Name one or more colors" with acceptable answers including red, orange, yellow, green, blue, and purple, and they wrote "red, green, and the center of a black hole" that would be marked as correct because it contains at least one string from the acceptable list. Similarly, "hammered metal" would be marked as correct because it contains the string "red" ([you can prevent this if you want](https://www.w3schools.com/jsref/jsref_regexp_begin.asp)). On the other hand "teal, scarlet, indigo" would be marked wrong.
+
+```
+What are some items?
+
+[[this text will never show up if they type a right answer and click "Check", only if they click the checkmark button to reveal the answer]]
+[[?]] Hint: The answers are like "item1", "item2", etc.
+<script>
+  let input = "@input".trim();
+  /item1|item2|item3|item4/i.test(input);
+</script>
+***
+<div class = "answer">
+
+Reiterate what the correct answer or answers should be, and try to anticipate likely wrong answers so you can explain why they're not correct.
+
+</div>
+***
+```
+
+What are some items?
+
+[[this text will never show up if they type a right answer and click "Check", only if they click the checkmark button to reveal the answer]]
+[[?]] Hint: The answers are like "item1", "item2", etc.
+<script>
+  let input = "@input".trim();
+  /item1|item2|item3|item4/i.test(input);
+</script>
+***
+<div class = "answer">
+
+Reiterate what the correct answer or answers should be, and try to anticipate likely wrong answers so you can explain why they're not correct.
+
+</div>
+***
+
+Note that you can use any markdown formatting you want in quizzes, including bold, links, math, lists, embedded media, code blocks, etc. 
+For accessibility for learners using screenreaders, if your question is more than a single markdown paragraph, you must enclose it in `<div>` tags. For example:
+
+```
+<div>
+In the following code block, what should fill in the blank?
+
+```r
+install.packages(____)
+```
+</div>
```

Comment:
Should this `</div>` be inside the code block?

Reply:
Ah, I messed up the nesting code blocks there, good catch! Fixed now.